frontend/src/scenes/settings/organization/InviteModal.tsx

```diff
@@ -124,69 +165,11 @@ export function InviteTeamMatesComponent(): JSX.Element {
             )}
             <div className="space-y-2">
                 <div className="flex gap-2">
-                    <b className="flex-1">Email address</b>
+                    <b className="flex-2">Email address</b>
                     <b className="flex-1">{preflight?.email_service_available ? 'Name (optional)' : 'Invite link'}</b>
+                    {allowedLevelsOptions.length > 1 && <b className="flex-1">Level</b>}
                 </div>
 
-                {invitesReversed.map((invite: OrganizationInviteType) => {
```

Comment:
To confirm this is just removing existing invites from the modal? Works for me.

Reply:
Yeah, I'm removing existing invites. I thought it looked cluttered and didn't think it added anything beyond the pending invites table already on the settings page.